docs/basics/compatible-controllers.md

```
@@ -128,6 +128,7 @@ ESP-01 | ESP8266 | One of the first and cheapest ESP8266 boards available. **_No
 ![Gold Member](../assets/images/content/akemi_gold.png)<br/> [ESP32 Mini Dev Board 16mb by SerKo (aka Serg)](https://www.tindie.com/products/28858/)| ESP32 | ESP32 Dev Board with latest WROOM-32E module, USB Type-C, PTC fused, over-voltage protection. D1 Mini32 form factor and compatible [pin out](https://github.com/srg74/WLED-wemos-shield/blob/master/resources/ESP32_mini/Resources/ESP32_mini_board_pinout.pdf)
 [H803 WiFi](https://github.com/srg74/WLED/wiki/H803WiFi) | ESP8266 | ESP8266EX based controller with level shifter inside. Data pin GPIO1 Clock pin GPIO14. Tested with WS2813 strip and [Firmware fork is here](https://github.com/srg74/WLED/tree/H803WF).
 [Heltec WiFi Kit 8](https://heltec.org/project/wifi-kit-8/) | ESP8266 | Another alternative of ESP8266 board. OLED display 128X32 pixel, battery charger on board. Almost the same functionality and price as the Wemos board. Plus it can be used in projects with external batteries.
+[Heltec WiFi LoRa 32 V3](https://heltec.org/project/wifi-lora-32-v3/) | ESP32-S3 | Compact board with ESP32-S3FN8, 8MB Flash, 320KB RAM (**NO PSRAM**), 0.96" SSD1306 OLED display, and LoRa 915MHz. **Must use `heltec_wifi_lora_32_v3` build environment.** Onboard I2C display (GPIO17 SDA, GPIO18 SCL, GPIO21 RST, GPIO36 Vext). Supports WiFi, Bluetooth, and display usermod. See compatible hardware section for OLED setup with FLD_PIN_RST and HELTEC_VEXT_PIN.
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)